src/arrays/creating_arrays/hard/q3/StoringInts.java

```diff
@@ -0,0 +1,30 @@
+package arrays.creating_arrays.hard.q3;
+
+/*
+Write a program that asks the user for 5 integers, and stores those values into an array and prints the array, with
```

Comment:
Change question text to:
Write a program that asks the user for five integers, stores those values into an array, and prints the array with each value on a new line back to the user.